godotopenxrmeta/src/main/cpp/include/openxr_fb_render_model.h

```diff
@@ -0,0 +1,69 @@
+/*  openxr_fb_render_model.h                                              */
+/**************************************************************************/
+/*                       This file is part of:                            */
+/*                              GODOT XR                                  */
+/*                      https://godotengine.org                           */
+/**************************************************************************/
+/* Copyright (c) 2022-present Godot XR contributors (see CONTRIBUTORS.md) */
+/*                                                                        */
+/* Original contributed implementation:                                   */
+/*   Copyright (c) 2022-2023 MattaKis Consulting Kft. (Migeran)           */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef OPENXR_FB_RENDER_MODEL_H
+#define OPENXR_FB_RENDER_MODEL_H
+
+#include <godot_cpp/classes/node3d.hpp>
+
+namespace godot {
+
+class OpenXRFbRenderModel : public Node3D {
+	GDCLASS(OpenXRFbRenderModel, Node3D)
+
+private:
+	String render_model_path;
+	Node3D *render_model = nullptr;
+
+	void load_render_model();
+
+protected:
+	void _notification(int p_what);
+
+	static void _bind_methods();
+
+	void _validate_property(PropertyInfo &p_property) const;
+
+public:
+	PackedStringArray get_suggested_render_model_path_names() const;
+
+	void set_render_model_path(String p_render_model_path);
```

Comment:
Let's remove this method. Through the api users should only have access to what we know is supported by the runtime.

Reply:
What do you mean by this? The user needs some way to select which model should be loaded. Do you mean switching from a `String` to an `enum` so they can only use certain values, rather than a free-form string?

Reply:
Yeah I was referring to remove users' ability to enter a path themselves, so switching to an `enum` would work.